### DISCUSSION TITLE
Apollo Save Tool v1.8.0

### DISCUSSION THREAD

**Discussion by bucanero** [bucanero/apollo-ps3#105] [General]
_dedicated to Leon ~ in loving memory (2009 - 2022)_ :heart:

![](https://github.com/bucanero/apollo-vita/raw/main/data/leon.jpg)

### Added

* Import Trophies from USB
* Improve Trophy bulk export
* Download Online DB saves to HDD
* Extract archive tool (`.zip`, `.7z`, `.rar`)
* Network Tools
  * URL downloader tool (download http/https/ftp/ftps links)
  * Simple local Web Server (full access to console drives)
* New save-game sorting options (`Settings`)
  * by Name, by Title ID
* Show PS3 IP address when running Apollo's Web Server
* Improve USB support (scan USB0 to USB6)
* On-screen Keyboard (for text input)

### Fixed

* Fix screen aspect ratio (Tiny3D viewport issue)
* Fix list refresh bug with `.PSV` saves
* Fix `.VM2` to `.VMC` export bug
* Fix bug when saving settings

### Misc

* Download application data updates from `apollo-patches` repository
* Use `.savepatch` extension for save-game patches
* Updated Apollo patch engine v0.3.0
  * BSD scripting changes
  * New custom hash: `mgspw_checksum`
  * Fix `sha1_xor64` custom hashing
  * Fix little-endian support for decrypters/hashes (PS4/Vita)

<hr /><em>This discussion was created from the release <a href='https://github.com/bucanero/apollo-ps3/releases/tag/v1.8.0'>Apollo Save Tool v1.8.0</a>.</em>

**Comment by Luscas23** [comment]
Hello I would like to know how do I use chets to save data

**Reply by bucanero** [reply]
If cheats are available for the game TITLE ID, they'll show up when you select the save from the list. Then you can select the codes you want, and "Apply changes & resign".

When a game has codes you'll see something like: ![](https://github.com/bucanero/apollo-ps3/raw/master/docs/screenshots/screenshot_patches.png)

**Comment by frankyfife** [comment]
I'm so sorry for your loss.

**Comment by DragonMaster17** [comment]
I was wondering about the dog - since i made the update i wanted to come here to see for myself. I'm sorry for your loss.
And about the program itself, is there a way to delete Trophy Files from the Console? The more Trophy files i get over time, the loonger it takes for Database Restruction.

**Comment by Zinoujoker** [comment]
Cool
